### PR TITLE
Feature/firestore

### PR DESCRIPTION
**Pull request by evilpenguin**

### CODE REVIEW

File: Say Their Names/Source/Log/Log.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
+//
+//  Log.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 5/31/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Log/Log.swift:9]: Delete duplicate please

File: Say Their Names/Source/Frameworks/ServiceFrameworks.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+// MARK: - ServiceFrameworks
+
+typealias ServiceFrameworks = Service
+extension ServiceFrameworks {
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Frameworks/ServiceFrameworks.swift:15]: Whats the benefit of using a `typealias` here?

Comment (evilpenguin) [reply Say Their Names/Source/Frameworks/ServiceFrameworks.swift:15]: There is none outside of abstraction, and not just have `extension class` everywhere.

File: Say Their Names/Source/Log/Log.swift (hunk 1 above)
Comment (evilpenguin) [reply Say Their Names/Source/Log/Log.swift:9]: Removed

File: Say Their Names/Source/Frameworks/ServiceFrameworks.swift (hunk 2 above)
Comment (Kilo-Loco) [reply Say Their Names/Source/Frameworks/ServiceFrameworks.swift:15]: If you think this makes things easier from a readability pov then let's go ahead and keep it. I'll leave it up to you

Comment (evilpenguin) [reply Say Their Names/Source/Frameworks/ServiceFrameworks.swift:15]: We can always remove it later. Simple refactor in the feature.

**Pull request merged**